scripts/get-changed-services/index.ts

```diff
@@ -28,11 +28,19 @@ async function main() {
     // get the latest commit in the branch to see if we are forcing a run
     const latestCommit = await getLatestCommitSHA()
     const commitMessage = await getLatestCommitMessage(latestCommit)
+
+    // force a complete CI run
     if (commitMessage.includes('ci-force-run')) {
         core.setOutput('changed-services', deployAllServices)
         return
     }
 
+    // force just a cypress re-run
+    if (commitMessage.includes('cypress re-run')) {
```

Comment:
I didn't know about ci-force-run, where would be a good place to document these CI incantations?

Reply:
The `force-ci-run` is in the main Readme, but it looks like it's a little buried and hard to pick out. I just pushed up a change that documents the new `cypress re-run` and breaks those docs out into a section so it's easier to run across!